pkgs/by-name/sc/schroot/debian-patches.nix

```diff
@@ -0,0 +1,70 @@
+# Generated by debian-patches.sh from debian-patches.txt
+let
+  prefix = "https://sources.debian.org/data/main/s/schroot/1.6.13-5/debian/patches";
```

Comment:
I think this download will eventually break (as soon as the version is updated at Debian).

Reply:
Would it be ok if I used salsa.debian.org instead? i. e. https://salsa.debian.org/debian/schroot/-/raw/debian/master/1.6.13-4/debian/patches

Reply:
If they are accessible without logging in, yes